### DISCUSSION TITLE
Does the model only use data for the home team to make predictions?

### DISCUSSION THREAD

**Discussion by rpassaro** [kyleskom/NBA-Machine-Learning-Sports-Betting#420] [Q&A]
Looking at the SQLite file, it appears as though the model only uses the Home Team data to train. Is this accurate, or am I missing something? Any help would be appreciated! Thank you!
<img width="708" alt="Screenshot 2024-03-13 163603" src="https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/61886601/6f073abf-88c8-40ab-a9ae-29f3ed2acfaf">

**Comment by Shanna3229** [comment]
Keep scrolling to the right - it has each match
![image](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/18405777/6c03fb88-92ab-47cf-b93a-648534a69692)

**Comment by whiskey1906** [comment]
can I get a copy of this ?

**Reply by Youngstud4321** [reply]
Just download a SQlite viewer and open the sqlite files in the Data folder
